src/Forms/ConfirmedPasswordField.php

```diff
@@ -255,7 +272,36 @@ public function getChildren()
     public function setCanBeEmpty($value)
     {
         $this->canBeEmpty = (bool)$value;
+        $this->updateRightTitle();
+        return $this;
+    }
```

Comment:
I don't think the right title is a good place for this.

I frankly don't think _any_ of this belongs in `ConfirmedPasswordField` since we're implementing something that should only apply to `Member` records, but if you insiste on keeping it here and have a good reason, then we at least shouldn't be applying messages like this to areas that can be overridden by developers accidentally.

Consider for example:
```php
ConfirmedPasswordField::create('MyPassword')->setCanBeEmpty(true)->setRightTitle('some title I want to be here')
```
In this scenario, the developer accidentally overwrote our message.

If you really really think this belongs in the field and not in `Member`, it should probably be included through the template or some other place where it's harder to overwrite accidentally.

Reply:
It kind of does belong here, since the 'generate a random password' functionality is in the field, though the function to actually generate that random password in Member

I've updated this so that it'll prefix the 'A random password will be generated if blank' message to right title, rather than overwriting the whole right title